nilearn/plotting/img_plotting.py

```diff
@@ -1333,10 +1333,8 @@ def plot_connectome_strength(adjacency_matrix, node_coords, node_size="auto",
                              cmap=None, output_file=None, display_mode="ortho",
                              figure=None, axes=None, title=None):
     """Plot connectome strength on top of the brain glass schematics.
-
```

Comment:
I wouldn't adjust this white space

Reply:
re the function itself: You'll want to add a `DeprecationWarning` on this function call, and a test that the deprecation warning was successfully raised. Happy to point to existing examples as it's useful !